2.9.0/done/cherrypicks.md

```diff
@@ -0,0 +1,48 @@
+# cherry picks
+## bc breaking
+## deprecation
+## new features
+## improvements
+### inductor
+- [Graph Partition] improve custom op output alias ([#163380](https://github.com/pytorch/pytorch/pull/163380))
+
+## bug fixes
+### cuda
+- [cuDNN][SDPA][submodule] Roll-back cuDNN frontend upgrade, update Meta registration ([#163265](https://github.com/pytorch/pytorch/pull/163265))
+
+### cpu (aarch64)
+- Fix the regression issue caused by non-arrch64 platforms not hitting the MKLDNN path ([#162778](https://github.com/pytorch/pytorch/pull/162778))
+
+### inductor
+- [Cherry Pick][Graph Partition] allow sharing default device context ([#163097](https://github.com/pytorch/pytorch/pull/163097))
+
+### inductor (aoti)
+- Revert "Make distributed modules importable even when backend not built ([#163024](https://github.com/pytorch/pytorch/pull/163024))
```

Comment:
rephrase this to something human-readable. I'm not sure this revert should be included if the original PR's landed though. Or if the original PR got reverted for the release, then this commit + the original PR commit should both be moved to not user facing.